javaee/impl/src/main/java/org/jboss/forge/addon/javaee/rest/ui/RestNewEndpointCommandImpl.java

```diff
@@ -121,4 +121,14 @@ public JavaClassSource decorateSource(UIExecutionContext context, Project projec
 
       return source;
    }
+
+   @Override
+   public UISelectMany<RestMethod> getMethods() {
```

Comment:
It looks like the Forge formatter is not being used in here? https://github.com/forge/core/blob/master/eclipse-code-formatter-profile.xml